setup.cfg

```diff
@@ -2,7 +2,6 @@
 name = deepchem
 author = Bharath Ramsundar, Evan Feinberg, Pande Lab Stanford
 summary = Deep-learning models for drug discovery and quantum chemistry
-description-file = README.md
```

Comment:
There was utf-8 in the README.md (ö in Schrödinger).  This caused issues in the install.  I took out the description file rather than deal with it.